### DISCUSSION TITLE
Add comment outside the scope of the PR

### DISCUSSION THREAD

**Discussion by djoubert-psee** [community/community#30434] [Pull Requests]
Hello, 

I would like to open comments outside the diff of the PR, because sometimes a change has not be propagated correctly.

Can we do that ? The + icon to open a comment looks limited to few lines around the diffs. 

Thanks for any help

**Comment by airtower-luna** [comment]
Well, there's the general review comment: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request#submitting-your-review

Maybe that works for you?

**Reply by djoubert-psee** [reply]
Thanks for your help. Unfortunately this is not addressing the code outside the scope of the diff. 
Also, some figures in this page are outdated as I don't get the same interface.

**Comment by vedal** [comment]
Seems the same as: https://github.com/isaacs/github/issues/284
